unstructured/ingest/v2/cli/utils.py

```diff
@@ -3,6 +3,7 @@
 import sys
 from dataclasses import fields, is_dataclass
 from gettext import gettext, ngettext
+from gettext import gettext as _
```

Comment:
could you explain this a little more. Its also imported above this line as gettext. Also `_` traditionally means unused variable.

Reply:
I simply copied this from the v1 file so there was no dependency on it anymore. I didn't write this code new as part of this PR.